ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Ds3Cli.java

```diff
@@ -104,7 +104,7 @@ public Ds3Cli(final Ds3Provider provider, final Arguments args, final FileUtils
         jsonViews.put(CommandValue.GET_JOB,                new com.spectralogic.ds3cli.views.json.GetJobView());
         jsonViews.put(CommandValue.DELETE_FOLDER,          deleteView);
         jsonViews.put(CommandValue.PUT_JOB,                new com.spectralogic.ds3cli.views.json.PutJobView());
-        jsonViews.put(CommandValue.GET_TAPES,              new com.spectralogic.ds3cli.views.json.GetTapesWithFullDetailsView());
+        jsonViews.put(CommandValue.GET_TAPES,              new com.spectralogic.ds3cli.views.json.GetTapesView());
```

Comment:
The details are needed.  I'd like to understand the problem more.

https://github.com/SpectraLogic/ds3_java_cli/pull/71